frontend/packages/react-app/src/components/organisms/TokenCampaignDetailPanel/index.tsx

```diff
@@ -30,15 +36,28 @@ const TokenDetailCampaignPanel: React.FC<TokenDetailCampaignPanelProps> = ({
 
   return (
     <div style={{ marginTop: "24px" }}>
+      {!audiusState.user && (
+        <SigninAudius
+          audiusState={audiusState}
+          audiusDispatch={audiusDispatch}
+        />
+      )}
       <TokenCampaignDetail campaign={campaign} />
-      <TokenRequestCard state={state} dispatch={dispatch} />
+      <TokenRequestCard
```

Comment:
You might want to check if user have an Audius login.

Reply:
IMO: Also if campaign not started yet, we shouldn't show a check request UI.

Reply:
Does this means user should be checked whether it is present or not?
`user` is checked inside component. 
https://github.com/TART-tokyo/iroiro/blob/ceb89422018aba3cff58f67ca3c567f1dc6c83ce/frontend/packages/react-app/src/components/molecules/TokenRequestCard/index.tsx#L67

Reply:
This is to avoid confusing the user and make them follow the steps.
In my opinion that the check request UI should be hidden when the user is not logged in to Audius.